lib/tito/event.rb

```diff
@@ -0,0 +1,44 @@
+class Tito
+  class Event
+    def initialize(details)
+      @details = details
+    end
+
+    def title
+      details["title"]
+    end
+
+    def description
+      content = details["description"]
+
+      if content.empty?
```

Comment:
I don’t believe this works if the description is an empty string. Or at least not in an IRB console.